uplink-sys/build.rs

```diff
@@ -29,11 +29,18 @@ fn main() {
 
     // Directory containing uplink-c project for building
     let uplink_c_dir = out_dir.join("uplink-c");
-    // Copy project to OUT_DIR for building
+
+    // Make the output `uplink-c` directory if it doesn't exist.
+    Command::new("mkdir")
+        .args(["-p", &uplink_c_dir.to_string_lossy()])
+        .status()
+        .expect("Failed to create output uplink-c directory.");
+
+    // Copy the contents of the `uplink-c` project to the output `uplink-c` directory
     Command::new("cp")
         .args([
             "-R",
```

Comment:
Using  `-f` will solve the described issue. Let's add `f` here (i.e. `-Rf`) and rollback all the other changes.